apps/landing/pages/index.tsx

```diff
@@ -1,5 +1,5 @@
 import HomeLayout from 'components/layouts/Home';
-import Developer from 'features/home/Developer';
+// import Developer from 'features/home/Developer';
```

Comment:
why did you not remove it instead of commenting?

Reply:
I only hide temporarily, because it has to wait for content update